rapids-cmake/cpm/thrust.cmake

```diff
@@ -56,17 +56,18 @@ Result Variables
 function(rapids_cpm_thrust NAMESPACE namespaces_name)
   list(APPEND CMAKE_MESSAGE_CONTEXT "rapids.cpm.thrust")
 
+  include("${rapids-cmake-dir}/cpm/detail/package_details.cmake")
+  rapids_cpm_package_details(Thrust version repository tag shallow exclude)
```

Comment:
Kinda confused as to why this moved, this information isn't being used anywhere new is it? The new lines outside of the conditional scope don't seem to use them, what am I missing?

Reply:
Nice eye. The issue is that it should be used in the logic to compute `to_install` and wasn't.
Update the PR to correct this bug.